source/guides/ssl-setup-nginx.rst

```diff
@@ -73,6 +91,13 @@ First install certbot. This is the tool we will be using to obtain the certifica
          
          sudo apt-get update
          sudo apt-get install certbot python3-certbot-nginx
+   
+   .. tab:: CentOS
+
+      .. code-block:: bash
+            
+            sudo dnf install epel-release
+            sudo dnf install certbot python3-certbot-nginx
```

Comment:
Just a question from someone not very familiar with CentOS: Is there any reason to use `yum` for one thing and `dnf` for another?  
Just feels odd to me to use multiple package managers

Reply:
they do the same stuff in my experience but im not expert

Reply:
@nepcore ive made changes based on your comments i hope they work

Reply:
fyi: i googled around a bit and apparently dnf is a yum alternative that resolves a few issues yum has had for a long time, so to me dnf does seem to be the better choice over all